src/DssSpell.sol

```diff
@@ -65,312 +72,282 @@ contract DssSpellAction is DssAction {
-        DssExecLib.setValue(DIRECT_SPARK_DAI_PLAN, "buffer", 100 * MILLION * WAD);
+        // Increase GSM Delay by 14 hours, from 16 hours to 30 hours
+        PauseLike(MCD_PAUSE).setDelay(30 hours);
 
-        // ---------- Update Gelato Keeper Treasury Address ----------
-        // Forum: https://forum.sky.money/t/gelato-keeper-update/25456
+        // ---------- Add ilks to the LineMom ----------
+        // Forum: https://forum.sky.money/t/standby-spells-for-sky-emergency-response/25594/2
 
-        // Update DssExecLib.setContract: GELATO_PAYMENT_ADAPTER - "treasury" to 0x5041c60C75633F29DEb2AED79cB0A9ed79202415
-        DssExecLib.setContract(GELATO_PAYMENT_ADAPTER, "treasury", GELATO_TREASURY_NEW);
+        // Add ALLOCATOR-SPARK-A to the LineMom
+        LineMomLike(LINE_MOM).addIlk("ALLOCATOR-SPARK-A");
 
-        // ---------- ConsolFreight Debt Write-Off and DAO Resolution ----------
-        // Forum: https://forum.sky.money/t/consolfreight-rwa-003-cf4-drop-default/21745/21
-        // Forum: https://forum.sky.money/t/consolfreight-rwa-003-cf4-drop-default/21745/22
+        // Add RWA001-A to the LineMom
+        LineMomLike(LINE_MOM).addIlk("RWA001-A");
 
-        // Account for the accumulated stability fee by calling jug.drip("RWA003-A")
-        JugAbstract(MCD_JUG).drip("RWA003-A");
+        // Add RWA002-A to the LineMom
+        LineMomLike(LINE_MOM).addIlk("RWA002-A");
 
-        // Write-off the debt of RWA003-A by calling rwaLiqOrcl.cull("RWA003-A", RWA003_A_URN)
-        RwaLiquidationOracleLike(MIP21_LIQUIDATION_ORACLE).cull("RWA003-A", RWA003_A_URN);
+        // Add RWA009-A to the LineMom
+        LineMomLike(LINE_MOM).addIlk("RWA009-A");
 
-        // Approve ConsolFreight Dao Resolution with IPFS hash QmX4DdVBiDBjLXYT4J4jC1XMdTn2Q7Ao8L66pKB8N3yETA
-        // Note: see `dao_resolutions` public variable declared above
+        // Add RWA012-A to the LineMom
+        LineMomLike(LINE_MOM).addIlk("RWA012-A");
 
-        // ---------- Sky Ecosystem Liquidity Bootstrapping Funding ----------
-        // Forum: https://forum.sky.money/t/utilization-of-the-sky-ecosystem-liquidity-bootstrapping-budget-a-5-6-1-9/25537
-        // Poll: https://vote.makerdao.com/polling/QmYHUDVA#poll-detail
+        // Add RWA013-A to the LineMom
+        LineMomLike(LINE_MOM).addIlk("RWA013-A");
+
+        // Add RWA015-A to the LineMom
+        LineMomLike(LINE_MOM).addIlk("RWA015-A");
+
+        // ---------- Add emergency spells to the chainlog ----------
+        // Forum: https://forum.sky.money/t/standby-spells-for-sky-emergency-response/25594
+        // Poll: https://vote.makerdao.com/polling/QmRfTQ5t
+
+        // Contract: SingleClipBreakerFactory - Key: EMSP_CLIP_BREAKER_FAB - Value: 0xd7321d0919573a33f9147fD2579a48F60237564A
+        DssExecLib.setChangelogAddress("EMSP_CLIP_BREAKER_FAB", 0xd7321d0919573a33f9147fD2579a48F60237564A);
```

Comment:
Please declare all addresses outside of the `actions` function instead of hardcoding them inside the function call

Reply:
Done